pom.xml

```diff
@@ -1,144 +1,44 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.liquibase</groupId>
+        <artifactId>liquibase-parent-pom</artifactId>
+        <version>0.1.0</version>
+    </parent>
     <groupId>org.liquibase.ext</groupId>
     <artifactId>liquibase-mongodb</artifactId>
     <packaging>jar</packaging>
     <version>4.23.2-SNAPSHOT</version>
     <name>Liquibase MongoDB Extension</name>
     <description>Liquibase Extension for MongoDB</description>
     <url>http://www.liquibase.org</url>
-    <organization>
-        <name>Liquibase.org</name>
-        <url>http://www.liquibase.org</url>
-    </organization>
-    <licenses>
-        <license>
-            <url>http://www.apache.org/licenses/LICENSE-2.0</url>
-            <name>Apache License, Version 2.0</name>
-        </license>
-    </licenses>
-    <issueManagement>
-        <url>https://github.com/liquibase/liquibase-mongodb/issues</url>
-    </issueManagement>
-    <scm>
-        <connection>scm:git:https://github.com/liquibase/liquibase-mongodb.git</connection>
-        <url>https://github.com/liquibase/liquibase-mongodb</url>
-        <tag>HEAD</tag>
-    </scm>
```

Comment:
Do you think this information should be removed? It seems the parent pom does not add it.

Reply:
it is here: https://github.com/liquibase/liquibase-parent-pom/blob/a0236c6084324ad3c70a5c162775c0969dfd8c55/pom.xml#L93

Reply:
I missed that, thank you. However, when I run `mvn clean install -DskipTests -Dbuild=release` locally, this information is not added to the result pom file. Hope it will be added on CI.

Reply:
yes, it will.